Sources/Testing/Events/Recorder/Event.AdvancedConsoleOutputRecorder.swift

```diff
@@ -0,0 +1,79 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+extension Event {
+  /// An experimental console output recorder that provides enhanced test result
+  /// display capabilities.
+  ///
+  /// This recorder is currently experimental and must be enabled via the
+  /// `SWT_ENABLE_EXPERIMENTAL_CONSOLE_OUTPUT` environment variable.
+  ///
+  /// Future capabilities will include:
```

Comment:
The documentation for this type should generally describe what it does, not what it will do someday in the future. There's no need to keep a to-do list in this comment; just describe the type in general terms.